charts/sonarqube/templates/deployment.yaml

```diff
@@ -162,7 +162,7 @@ spec:
               awk 1 /tmp/props/sonar.properties /tmp/props/secret.properties > /tmp/result/sonar.properties
             fi
           volumeMounts:
-          {{- if .Values.sonarProperties }}
```

Comment:
I think there is a condition missing here at line [146](https://github.com/SonarSource/helm-chart-sonarqube/pull/261/files#diff-d8492fb6e757528994fc1f4a506d435308d214cf5ed5903c9c9f6058ee14e12eL146)
The block will not be added if we only have a sonarSecretKey

Reply:
Good catch, changed things accordingly